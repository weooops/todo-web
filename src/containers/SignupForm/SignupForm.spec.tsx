// import * as React from 'react';
// import { shallow } from 'enzyme';

// import SignupForm from './SignupForm';

// it('renders correctly', () => {
//   const props = {
//     username: '',
//     email: '',
//     password: '',
//     comparePassword: '',
//     handleInputChange: jest.fn(),
//     handleSubmit: jest.fn()
//   };
//   const wrapper = shallow(<SignupForm {...props} />);
//   expect(wrapper).toMatchSnapshot();
// });
