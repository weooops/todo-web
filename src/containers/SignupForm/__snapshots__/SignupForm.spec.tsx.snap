// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <LoginForm
    comparePassword=""
    email=""
    handleInputChange={[Function]}
    handleSubmit={[Function]}
    password=""
    username=""
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <form
        onSubmit={[Function]}
      >
        <input
          name="username"
          onChange={[Function]}
          placeholder="Username"
          type="text"
          value=""
        />
        <input
          name="email"
          onChange={[Function]}
          placeholder="Email"
          type="text"
          value=""
        />
        <input
          name="password"
          onChange={[Function]}
          placeholder="Password"
          type="password"
          value=""
        />
        <input
          name="comparePassword"
          onChange={[Function]}
          placeholder="Password Confirm"
          type="password"
          value=""
        />
        <input
          type="submit"
          value="Sign up"
        />
      </form>,
      "className": "signup-form",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <input
            name="username"
            onChange={[Function]}
            placeholder="Username"
            type="text"
            value=""
          />,
          <input
            name="email"
            onChange={[Function]}
            placeholder="Email"
            type="text"
            value=""
          />,
          <input
            name="password"
            onChange={[Function]}
            placeholder="Password"
            type="password"
            value=""
          />,
          <input
            name="comparePassword"
            onChange={[Function]}
            placeholder="Password Confirm"
            type="password"
            value=""
          />,
          <input
            type="submit"
            value="Sign up"
          />,
        ],
        "onSubmit": [Function],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "name": "username",
            "onChange": [Function],
            "placeholder": "Username",
            "type": "text",
            "value": "",
          },
          "ref": null,
          "rendered": null,
          "type": "input",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "name": "email",
            "onChange": [Function],
            "placeholder": "Email",
            "type": "text",
            "value": "",
          },
          "ref": null,
          "rendered": null,
          "type": "input",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "name": "password",
            "onChange": [Function],
            "placeholder": "Password",
            "type": "password",
            "value": "",
          },
          "ref": null,
          "rendered": null,
          "type": "input",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "name": "comparePassword",
            "onChange": [Function],
            "placeholder": "Password Confirm",
            "type": "password",
            "value": "",
          },
          "ref": null,
          "rendered": null,
          "type": "input",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "type": "submit",
            "value": "Sign up",
          },
          "ref": null,
          "rendered": null,
          "type": "input",
        },
      ],
      "type": "form",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <form
          onSubmit={[Function]}
        >
          <input
            name="username"
            onChange={[Function]}
            placeholder="Username"
            type="text"
            value=""
          />
          <input
            name="email"
            onChange={[Function]}
            placeholder="Email"
            type="text"
            value=""
          />
          <input
            name="password"
            onChange={[Function]}
            placeholder="Password"
            type="password"
            value=""
          />
          <input
            name="comparePassword"
            onChange={[Function]}
            placeholder="Password Confirm"
            type="password"
            value=""
          />
          <input
            type="submit"
            value="Sign up"
          />
        </form>,
        "className": "signup-form",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <input
              name="username"
              onChange={[Function]}
              placeholder="Username"
              type="text"
              value=""
            />,
            <input
              name="email"
              onChange={[Function]}
              placeholder="Email"
              type="text"
              value=""
            />,
            <input
              name="password"
              onChange={[Function]}
              placeholder="Password"
              type="password"
              value=""
            />,
            <input
              name="comparePassword"
              onChange={[Function]}
              placeholder="Password Confirm"
              type="password"
              value=""
            />,
            <input
              type="submit"
              value="Sign up"
            />,
          ],
          "onSubmit": [Function],
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "name": "username",
              "onChange": [Function],
              "placeholder": "Username",
              "type": "text",
              "value": "",
            },
            "ref": null,
            "rendered": null,
            "type": "input",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "name": "email",
              "onChange": [Function],
              "placeholder": "Email",
              "type": "text",
              "value": "",
            },
            "ref": null,
            "rendered": null,
            "type": "input",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "name": "password",
              "onChange": [Function],
              "placeholder": "Password",
              "type": "password",
              "value": "",
            },
            "ref": null,
            "rendered": null,
            "type": "input",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "name": "comparePassword",
              "onChange": [Function],
              "placeholder": "Password Confirm",
              "type": "password",
              "value": "",
            },
            "ref": null,
            "rendered": null,
            "type": "input",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "type": "submit",
              "value": "Sign up",
            },
            "ref": null,
            "rendered": null,
            "type": "input",
          },
        ],
        "type": "form",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
